policy/modules/kernel/devices.if

```diff
@@ -4778,6 +4868,24 @@ interface(`dev_create_urand_dev',`
 	create_chr_files_pattern($1, device_t, urandom_device_t)
 ')
 
+########################################
+## <summary>
+##  Mount on the urandom device (/dev/urandom).
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`dev_mounton_urand_dev',`
```

Comment:
Lots of mounts. Seems like this warrants something like `init_mountpoint()`

Reply:
OK. For all the new interfaces I added for this access, I've removed those interfaces and instead made these types `container_mountpoint_type`.